subsys/net/lib/wifi_provision/proto/common.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+enum Band {
+  BAND_UNSPECIFIED = 0;
+  BAND_2_4_GHZ = 1;
+  BAND_5_GHZ = 2;
+  BAND_6_GHZ = 3;
```

Comment:
I see that SoftAp adds support for 6 ghz, but should this be reflected on the ble example as well?

Reply:
Probably yes.

Reply:
Removed 6ghz entry